generic_lambda/main.tf

```diff
@@ -0,0 +1,95 @@
+resource "aws_lambda_function" "lambda_function" {
+  function_name = var.function_name
+  handler       = var.handler
+  role          = aws_iam_role.lambda_iam_role.arn
+  runtime       = var.runtime
+  filename      = startswith(var.runtime, "java") ? "${path.module}/functions/generic.jar" : "${path.module}/functions/generic.zip"
+  timeout       = var.timeout_seconds
+  memory_size   = var.memory_size
+
+  ephemeral_storage {
+    size = var.storage_size
+  }
+
+  reserved_concurrent_executions = var.reserved_concurrency
+  tags                           = var.common_tags
```

Comment:
Maybe rename this var as just `tags`given they're just the tags for the lambda now, and not a set of common ones

Reply:
Good point, I've changed it.